gulpfile.js

```diff
@@ -224,7 +223,8 @@ gulp.task("plugins", function() {
     }
 
     return stream
-
+        .pipe(browsersync.reload({stream: true}))
+        .pipe(gulp.dest(JS_DIR.replace(SOURCE_DIR, out_dir)))
```

Comment:
I think these should be in the reverse order: first replacing the file and then reloading the browser.